app.js

```diff
@@ -42,57 +62,163 @@ app.post('/sync-from-kbo/:kboStructuredIdUuid', async function(req, res) {
+    adressComponent: (typeof adressComponent != 'undefined' ? adressComponent : ""),
+  };
+}
+
+new CronJob(
+  CRON_PATTERN,
+  async function () {
+    const now = new Date().toISOString();
+    console.log(`OVO numbers healing triggered by cron job at ${now}`);
+    try {
+      await healOvoNumbers();
+    } catch (err) {
+      console.log(
+        `An error occurred during OVO numbers healing at ${now}: ${err}`
+      );
+    }
+  },
+  null,
+  true
+);
 
 async function healOvoNumbers() {
   try {
-    console.log('Healing starting...');
+    console.log("Healing starting...");
     const identifiersCouplesOP = await getAllOvoAndKboCouples();
     const identifiersCouplesWegwijs = await getAllOvoAndKboCouplesWegwijs();
-  
+
     for (const identifiersCoupleOP of identifiersCouplesOP) {
       const wegwijsOvo = identifiersCouplesWegwijs[identifiersCoupleOP.kbo];
       // If a KBO can't be found in wegwijs but we already have an OVO for it in OP, we keep that OVO.
       // It happens especially a lot for worship services that sometimes lack data in Wegwijs
-      if (wegwijsOvo && (identifiersCoupleOP.ovo != wegwijsOvo)) {
+      if (wegwijsOvo && identifiersCoupleOP.ovo != wegwijsOvo) {
         // We have a mismatch, let's update the OVO number
         let ovoStructuredIdUri = identifiersCoupleOP.ovoStructuredId;
         if (!ovoStructuredIdUri) {
-          ovoStructuredIdUri = await constructOvoStructure(identifiersCoupleOP.kboStructuredId);
+          ovoStructuredIdUri = await constructOvoStructure(
+            identifiersCoupleOP.kboStructuredId
+          );
         }
 
         await updateOvoNumberAndUri(ovoStructuredIdUri, wegwijsOvo);
       }
     }
-    console.log('Healing complete!');
+    console.log("Healing complete!");
   } catch (err) {
-    console.log(`An error occurred during OVO numbers healing: ${err}`)
+    console.log(`An error occurred during OVO numbers healing: ${err}`);
   }
 }
 
 async function getAllOvoAndKboCouplesWegwijs() {
   let couples = {};
 
-  const response = await fetch(`https://api.wegwijs.vlaanderen.be/v1/search/organisations?q=kboNumber:/.*[0-9].*/&fields=ovoNumber,kboNumber&scroll=true`);
-  const scrollId = JSON.parse(response.headers.get('x-search-metadata')).scrollId;
+  const response = await fetch(
+    `https://api.wegwijs.vlaanderen.be/v1/search/organisations?q=kboNumber:/.*[0-9].*/&fields=ovoNumber,kboNumber&scroll=true`
+  );
+  const scrollId = JSON.parse(
+    response.headers.get("x-search-metadata")
+  ).scrollId;
   let data = await response.json();
 
   do {
-    data.forEach(unit => {
+    data.forEach((unit) => {
       couples[unit.kboNumber] = unit.ovoNumber;
     });
 
-    const response = await fetch(`https://api.wegwijs.vlaanderen.be/v1/search/organisations/scroll?id=${scrollId}`);
+    const response = await fetch(
+      `https://api.wegwijs.vlaanderen.be/v1/search/organisations/scroll?id=${scrollId}`
```

Comment:
The first part of this URL occurs three times in this file, I would define that as a constant instead.